django/db/models/expressions.py

```diff
@@ -831,6 +846,13 @@ def as_sql(self, compiler, connection):
         return '*', []
 
 
+class DB_DEFAULT(Expression):
+    """Placeholder expression for the database default in an insert query."""
+
+    def as_sql(self, compiler, connection):
+        return 'DEFAULT', []
```

Comment:
This allows bulk inserts to work.

Unfortunately this isn't supported on sqlite, so we'll need a different approach there. My old approach allows single inserts to work, so maybe the best we can do is some combination of both approaches and a note documenting the sqlite restriction.

Reply:
See also: https://github.com/django/django/pull/13709#discussion_r646148688

Reply:
https://stackoverflow.com/questions/58695302/how-to-insert-rows-in-sqlite-using-default-values

Reply:
I've added back the old approach behind a new database feature, so sqlite now passes again for single inserts.